### PR TITLE
Update EducationStemAutomatedDecision to have email sent timestamps

### PR DESCRIPTION
**Pull request by zurbergram**
## Description of change

Add new columns to `education_stem_automated_decision` table and usages of columns.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#21522

### CODE REVIEW

**Review by dmccurdybah — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

File: config/sidekiq_scheduler.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ EducationForm::Create10203SpoolSubmissionsReport:
   description: "Send the daily 10203 report to the call center about spool file submissions"
 
 EducationForm::Create10203ApplicantDecisionLetters:
-  cron: "45 4 * * MON-FRI America/New_York"
+  every: '2m'
```
Comment (thilton-oddball) [inline config/sidekiq_scheduler.yml:27]: This is going from running M-F once a day to running every 2 minutes? That seems like quite a jump. Just want to verify that this is intended.

File: db/migrate/20210315162119_add_email_sent_at_to_education_stem_automated_decision.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class AddEmailSentAtToEducationStemAutomatedDecision < ActiveRecord::Migration[6.0]
```
Comment (thilton-oddball) [inline db/migrate/20210315162119_add_email_sent_at_to_education_stem_automated_decision.rb:1]: This migration and the schema change will need to be removed to their own PR. Dangerbot won't allow any other changes to be included with migrations.

File: db/schema.rb [hunk 3]
```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_03_12_211214) do
+ActiveRecord::Schema.define(version: 2021_03_15_162119) do
```
Comment (thilton-oddball) [inline db/schema.rb:13]: As mentioned above, this will need to be in a separate PR with the migration.

**Review by thilton-oddball — CHANGES_REQUESTED** [review]
One question on the scheduler and then the migration and schema change need to be removed from this PR and placed in a separate PR.

File: config/sidekiq_scheduler.yml (hunk 1 above)
Comment (zurbergram) [reply config/sidekiq_scheduler.yml:27]: Updating frequency for UAT testing. This change will be reverted after UAT testing

File: db/migrate/20210315162119_add_email_sent_at_to_education_stem_automated_decision.rb (hunk 2 above)
Comment (zurbergram) [reply db/migrate/20210315162119_add_email_sent_at_to_education_stem_automated_decision.rb:1]: https://github.com/department-of-veterans-affairs/vets-api/pull/6176

File: db/schema.rb (hunk 3 above)
Comment (zurbergram) [reply db/schema.rb:13]: https://github.com/department-of-veterans-affairs/vets-api/pull/6176

**Review by thilton-oddball — APPROVED** [review]
All looks good
